microcloud/cmd/microcloud/ask.go

```diff
@@ -691,6 +692,22 @@ func (c *CmdControl) askNetwork(sh *service.Handler, systems map[string]InitSyst
 					return nil
 				}
 
+				commaValidator := func(s string) error {
```

Comment:
@gabrielmougard can you use `validate.Optional(validate.IsListOf(validate.IsNetworkAddress)),` the same as LXD does here?